pilot/pkg/model/jwks_resolver.go

```diff
@@ -168,6 +168,7 @@ func newJwksResolverWithCABundlePaths(evictionDuration, refreshInterval, retryIn
 		httpClient: &http.Client{
 			Timeout: jwksHTTPTimeOutInSec * time.Second,
 			Transport: &http.Transport{
+				Proxy:             http.ProxyFromEnvironment,
```

Comment:
Is this the only place we need this? I guess its probably only needed for external calls and this is the only one of those?

Reply:
Yes, since fetching the jwt is only external call required.